scopes/component/ui/compare/version-picker/component-compare-version-picker.module.scss

```diff
@@ -25,6 +25,8 @@
   min-width: 0;
   align-items: center;
   font-size: 14px;
+  // TODO - create z-index pallete system.
+  z-index: $nav-z-index; // to prevent it from going over the responsive navbar dropdown, but still keep it above the content below.
```

Comment:
the version dropdown itself should fix this z-index 
this I believe just overrides it for compare
if we use the version dropdown somewhere else we will have the same problem

Instead in `version-dropdown.module.scss`"

we should fix the z-index to be on the menu and not the entire dropdown